monai/inferers/merger.py

```diff
@@ -257,15 +258,15 @@ def __init__(
             shape=self.merged_shape,
             chunks=self.chunks,
             dtype=self.value_dtype,
-            compressor=self.compressor,
+            compressor=None,
```

Comment:
Having a light compression is usually advantageous even from speed point of view.
https://news.ycombinator.com/item?id=25926758